tests/artifacts/test_artifacts.py

```diff
@@ -136,3 +139,65 @@ def test_load_image_invalid_image(run_with_text_artifact):
         mlflow.exceptions.MlflowException, match="Unable to form a PIL Image object"
     ):
         mlflow.artifacts.load_image(artifact.uri)
+
+
+@pytest.fixture()
+def text_artifact(tmp_path):
+    artifact_name = "test.txt"
+    artifacts_root_tmp = mkdir(tmp_path.joinpath(str(uuid.uuid4())))
+    test_artifact_path = artifacts_root_tmp.joinpath(artifact_name)
+    with open(test_artifact_path, "w") as handle:
+        handle.write("test")
+    artifact_return_type = namedtuple(
+        "artifact_return_type", ["tmp_path", "artifact_path", "artifact_name"]
+    )
+    yield artifact_return_type(artifacts_root_tmp, test_artifact_path, artifact_name)
+    shutil.rmtree(artifacts_root_tmp, ignore_errors=True)
```

Comment:
```suggestion
    return artifact_return_type(artifacts_root_tmp, test_artifact_path, artifact_name)
```

Do we need this line? pytest automatically cleans up temporary directories created by the `tmp_path` fixture.